.github/workflows/clang-format-fix.yml

```diff
@@ -1,25 +1,18 @@
 name: clang-format Check
-on: 
-  workflow_dispatch:
-  push:
+on:
+  pull_request:
 jobs:
   formatting-check:
     name: Formatting Check
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'skip-ci')"
     steps:
     - uses: actions/checkout@v2
-    - name: Run clang-format style check for C programs.
-      uses: DoozyX/clang-format-lint-action@v0.11
+    - name: Run clang-format style check for C and Java programs.
+      uses: DoozyX/clang-format-lint-action@v0.13
       with:
         source: '.'
-        extensions: 'c,h,cpp,hpp'
-        clangFormatVersion: 10
-        inplace: True
+        extensions: 'c,h,cpp,hpp,java'
+        clangFormatVersion: 13
```

Comment:
develop, 1.12 and 1.8 PRs have a line "inplace:  True".  Should that be here for 1.10 as well?

Reply:
probably

Reply:
yes that file got the wrong changes

Reply:
When this is fixed we can merge this